src/element.ohlc.js

```diff
@@ -0,0 +1,49 @@
+﻿'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	var globalOpts = Chart.defaults.global;
+
+	globalOpts.elements.ohlc = Object.assign(globalOpts.elements.financial, {
+		lineWidth: 2,
+		armLength: null,
+		armLengthRatio: 0.90,
+	});
+
+	Chart.elements.ohlc = Chart.elements.financial.extend({
+		draw: function() {
+			var ctx = this._chart.ctx;
+			var vm = this._view;
+
+			var x = vm.x;
+			var o = vm.candle.o;
+			var h = vm.candle.h;
+			var l = vm.candle.l;
+			var c = vm.candle.c;
+			var armLength = helpers.getValueOrDefault(vm.armLength, globalOpts.elements.ohlc.armLength);
+			var armLengthRatio = helpers.getValueOrDefault(vm.armLengthRatio, globalOpts.elements.ohlc.armLengthRatio);
+			if(armLength === null) {
+				armLength = vm.width * 0.7 * armLengthRatio;
```

Comment:
`0.7 * armLengthRatio`? I would expect the `armLengthRatio` to be smaller if you want the arm length smaller, so I'm not sure what the 0.7 is needed

Reply:
See https://github.com/chartjs/chartjs-chart-financial/pull/22/commits/683692ea80ce61d54ba5d71d546727a5cc061eb7

Reply:
Can you take that commit message and make it a code comment here? Otherwise it will be hard for people to understand that just from reading the code

Reply:
done